requirements.txt

```diff
@@ -2,4 +2,4 @@ astroid==2.11.5
 Django==3.2.14
 pre-commit==2.12.0
 pytest==6.1.1
-pylint==2.13.9
+pylint==3.3.1
```

Comment:
the policy needs to be ignored to fix the snyk error
not opposed to the upgrade